packages/components/select-v2/src/useSelect.ts

```diff
@@ -805,7 +805,7 @@ const useSelect = (props: ISelectV2Props, emit) => {
     () => {
       const input = inputRef.value
       // filter or remote-search scenarios are not initialized
-      if (!input || (input && document.activeElement !== input)) {
+      if (!input || !(props.filterable || props.remote)) {
```

Comment:
Why delete `(input && document.activeElement !== input)` ?

[Bug](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtc2VsZWN0LXYyXG4gICAgdi1tb2RlbD1cInZhbHVlXCJcbiAgICBzdHlsZT1cIndpZHRoOiAyNDBweFwiXG4gICAgbXVsdGlwbGVcbiAgICBmaWx0ZXJhYmxlXG4gICAgcmVtb3RlXG4gICAgOnJlbW90ZS1tZXRob2Q9XCJyZW1vdGVNZXRob2RcIlxuICAgIGNsZWFyYWJsZVxuICAgIDpvcHRpb25zPVwib3B0aW9uc1wiXG4gICAgOmxvYWRpbmc9XCJsb2FkaW5nXCJcbiAgICBwbGFjZWhvbGRlcj1cIlBsZWFzZSBlbnRlciBhIGtleXdvcmRcIlxuICAvPlxuICA8ZWwtYnV0dG9uIEBjbGljaz1cImhhbmRsZUFkZFwiPkNsaWNrIG1lPC9lbC1idXR0b24+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuXG5jb25zdCBzdGF0ZXMgPSBbXG4gICdBbGFiYW1hJyxcbiAgJ0FsYXNrYScsXG4gICdBcml6b25hJyxcbiAgJ0Fya2Fuc2FzJyxcbiAgJ0NhbGlmb3JuaWEnLFxuICAnQ29sb3JhZG8nLFxuICAnQ29ubmVjdGljdXQnLFxuICAnRGVsYXdhcmUnLFxuICAnRmxvcmlkYScsXG4gICdHZW9yZ2lhJyxcbiAgJ0hhd2FpaScsXG4gICdJZGFobycsXG4gICdJbGxpbm9pcycsXG4gICdJbmRpYW5hJyxcbiAgJ0lvd2EnLFxuICAnS2Fuc2FzJyxcbiAgJ0tlbnR1Y2t5JyxcbiAgJ0xvdWlzaWFuYScsXG4gICdNYWluZScsXG4gICdNYXJ5bGFuZCcsXG4gICdNYXNzYWNodXNldHRzJyxcbiAgJ01pY2hpZ2FuJyxcbiAgJ01pbm5lc290YScsXG4gICdNaXNzaXNzaXBwaScsXG4gICdNaXNzb3VyaScsXG4gICdNb250YW5hJyxcbiAgJ05lYnJhc2thJyxcbiAgJ05ldmFkYScsXG4gICdOZXcgSGFtcHNoaXJlJyxcbiAgJ05ldyBKZXJzZXknLFxuICAnTmV3IE1leGljbycsXG4gICdOZXcgWW9yaycsXG4gICdOb3J0aCBDYXJvbGluYScsXG4gICdOb3J0aCBEYWtvdGEnLFxuICAnT2hpbycsXG4gICdPa2xhaG9tYScsXG4gICdPcmVnb24nLFxuICAnUGVubnN5bHZhbmlhJyxcbiAgJ1Job2RlIElzbGFuZCcsXG4gICdTb3V0aCBDYXJvbGluYScsXG4gICdTb3V0aCBEYWtvdGEnLFxuICAnVGVubmVzc2VlJyxcbiAgJ1RleGFzJyxcbiAgJ1V0YWgnLFxuICAnVmVybW9udCcsXG4gICdWaXJnaW5pYScsXG4gICdXYXNoaW5ndG9uJyxcbiAgJ1dlc3QgVmlyZ2luaWEnLFxuICAnV2lzY29uc2luJyxcbiAgJ1d5b21pbmcnLFxuXVxuY29uc3QgbGlzdCA9IHN0YXRlcy5tYXAoKGl0ZW0pOiBMaXN0SXRlbSA9PiB7XG4gIHJldHVybiB7IHZhbHVlOiBgdmFsdWU6JHtpdGVtfWAsIGxhYmVsOiBgbGFiZWw6JHtpdGVtfWAgfVxufSlcblxuaW50ZXJmYWNlIExpc3RJdGVtIHtcbiAgdmFsdWU6IHN0cmluZ1xuICBsYWJlbDogc3RyaW5nXG59XG5cbmNvbnN0IHZhbHVlID0gcmVmKFsndmFsdWU6VGV4YXMnXSlcbmNvbnN0IG9wdGlvbnMgPSByZWY8TGlzdEl0ZW1bXT4oW10pXG5jb25zdCBsb2FkaW5nID0gcmVmKGZhbHNlKVxuXG5jb25zdCByZW1vdGVNZXRob2QgPSAocXVlcnk6IHN0cmluZykgPT4ge1xuICBpZiAocXVlcnkgIT09ICcnKSB7XG4gICAgbG9hZGluZy52YWx1ZSA9IHRydWVcbiAgICBzZXRUaW1lb3V0KCgpID0+IHtcbiAgICAgIGxvYWRpbmcudmFsdWUgPSBmYWxzZVxuICAgICAgb3B0aW9ucy52YWx1ZSA9IGxpc3QuZmlsdGVyKChpdGVtKSA9PiB7XG4gICAgICAgIHJldHVybiBpdGVtLmxhYmVsLnRvTG93ZXJDYXNlKCkuaW5jbHVkZXMocXVlcnkudG9Mb3dlckNhc2UoKSlcbiAgICAgIH0pXG4gICAgfSwgMjAwKVxuICB9IGVsc2Uge1xuICAgIG9wdGlvbnMudmFsdWUgPSBbXVxuICB9XG59XG5cbmZ1bmN0aW9uIGhhbmRsZUFkZCgpIHtcbiAgb3B0aW9ucy52YWx1ZSA9IGxpc3Rcbn1cbjwvc2NyaXB0PlxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctMTczNjItZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXSxcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWUsXG4gICAgXCJhbGxvd0pzXCI6IHRydWUsXG4gICAgXCJjaGVja0pzXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5pbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgY29uc3Qgc3R5bGVzID0gWydodHRwczovL3ByZXZpZXctMTczNjItZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnLCAnaHR0cHM6Ly9wcmV2aWV3LTE3MzYyLWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ10ubWFwKChzdHlsZSkgPT4ge1xuICAgIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgICAgbGluay5ocmVmID0gc3R5bGVcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgICB9KVxuICB9KVxuICByZXR1cm4gUHJvbWlzZS5hbGwoc3R5bGVzKVxufSIsIl9vIjp7InNob3dIaWRkZW4iOnRydWUsInN0eWxlU291cmNlIjoiaHR0cHM6Ly9wcmV2aWV3LTE3MzYyLWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)

Reply:
I think..., `(input && document.activeElement !== input)` is used to judge filter or remote-search scenarios, but #16714 caused the focus behavior of el-select-v2, so, i replaced `(input && document.activeElement !== input)` with `!(props.filterable || props.remote)`, might it have other uses? I have added it back, could you please check if the bug you mentioned has been fixed?

Reply:
It seems that the bug has not been solved yet

-[Before](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtc2VsZWN0LXYyXG4gICAgdi1tb2RlbD1cInZhbHVlXCJcbiAgICBzdHlsZT1cIndpZHRoOiAyNDBweFwiXG4gICAgbXVsdGlwbGVcbiAgICBmaWx0ZXJhYmxlXG4gICAgcmVtb3RlXG4gICAgOnJlbW90ZS1tZXRob2Q9XCJyZW1vdGVNZXRob2RcIlxuICAgIGNsZWFyYWJsZVxuICAgIDpvcHRpb25zPVwib3B0aW9uc1wiXG4gICAgOmxvYWRpbmc9XCJsb2FkaW5nXCJcbiAgICBwbGFjZWhvbGRlcj1cIlBsZWFzZSBlbnRlciBhIGtleXdvcmRcIlxuICAvPlxuICA8ZWwtYnV0dG9uIEBjbGljaz1cImhhbmRsZUFkZFwiPkNsaWNrIG1lPC9lbC1idXR0b24+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuXG5jb25zdCBzdGF0ZXMgPSBbXG4gICdBbGFiYW1hJyxcbiAgJ0FsYXNrYScsXG4gICdBcml6b25hJyxcbiAgJ0Fya2Fuc2FzJyxcbiAgJ0NhbGlmb3JuaWEnLFxuICAnQ29sb3JhZG8nLFxuICAnQ29ubmVjdGljdXQnLFxuICAnRGVsYXdhcmUnLFxuICAnRmxvcmlkYScsXG4gICdHZW9yZ2lhJyxcbiAgJ0hhd2FpaScsXG4gICdJZGFobycsXG4gICdJbGxpbm9pcycsXG4gICdJbmRpYW5hJyxcbiAgJ0lvd2EnLFxuICAnS2Fuc2FzJyxcbiAgJ0tlbnR1Y2t5JyxcbiAgJ0xvdWlzaWFuYScsXG4gICdNYWluZScsXG4gICdNYXJ5bGFuZCcsXG4gICdNYXNzYWNodXNldHRzJyxcbiAgJ01pY2hpZ2FuJyxcbiAgJ01pbm5lc290YScsXG4gICdNaXNzaXNzaXBwaScsXG4gICdNaXNzb3VyaScsXG4gICdNb250YW5hJyxcbiAgJ05lYnJhc2thJyxcbiAgJ05ldmFkYScsXG4gICdOZXcgSGFtcHNoaXJlJyxcbiAgJ05ldyBKZXJzZXknLFxuICAnTmV3IE1leGljbycsXG4gICdOZXcgWW9yaycsXG4gICdOb3J0aCBDYXJvbGluYScsXG4gICdOb3J0aCBEYWtvdGEnLFxuICAnT2hpbycsXG4gICdPa2xhaG9tYScsXG4gICdPcmVnb24nLFxuICAnUGVubnN5bHZhbmlhJyxcbiAgJ1Job2RlIElzbGFuZCcsXG4gICdTb3V0aCBDYXJvbGluYScsXG4gICdTb3V0aCBEYWtvdGEnLFxuICAnVGVubmVzc2VlJyxcbiAgJ1RleGFzJyxcbiAgJ1V0YWgnLFxuICAnVmVybW9udCcsXG4gICdWaXJnaW5pYScsXG4gICdXYXNoaW5ndG9uJyxcbiAgJ1dlc3QgVmlyZ2luaWEnLFxuICAnV2lzY29uc2luJyxcbiAgJ1d5b21pbmcnLFxuXVxuY29uc3QgbGlzdCA9IHN0YXRlcy5tYXAoKGl0ZW0pOiBMaXN0SXRlbSA9PiB7XG4gIHJldHVybiB7IHZhbHVlOiBgdmFsdWU6JHtpdGVtfWAsIGxhYmVsOiBgbGFiZWw6JHtpdGVtfWAgfVxufSlcblxuaW50ZXJmYWNlIExpc3RJdGVtIHtcbiAgdmFsdWU6IHN0cmluZ1xuICBsYWJlbDogc3RyaW5nXG59XG5cbmNvbnN0IHZhbHVlID0gcmVmKFsndmFsdWU6VGV4YXMnXSlcbmNvbnN0IG9wdGlvbnMgPSByZWY8TGlzdEl0ZW1bXT4oW10pXG5jb25zdCBsb2FkaW5nID0gcmVmKGZhbHNlKVxuXG5jb25zdCByZW1vdGVNZXRob2QgPSAocXVlcnk6IHN0cmluZykgPT4ge1xuICBpZiAocXVlcnkgIT09ICcnKSB7XG4gICAgbG9hZGluZy52YWx1ZSA9IHRydWVcbiAgICBzZXRUaW1lb3V0KCgpID0+IHtcbiAgICAgIGxvYWRpbmcudmFsdWUgPSBmYWxzZVxuICAgICAgb3B0aW9ucy52YWx1ZSA9IGxpc3QuZmlsdGVyKChpdGVtKSA9PiB7XG4gICAgICAgIHJldHVybiBpdGVtLmxhYmVsLnRvTG93ZXJDYXNlKCkuaW5jbHVkZXMocXVlcnkudG9Mb3dlckNhc2UoKSlcbiAgICAgIH0pXG4gICAgfSwgMjAwKVxuICB9IGVsc2Uge1xuICAgIG9wdGlvbnMudmFsdWUgPSBbXVxuICB9XG59XG5cbmZ1bmN0aW9uIGhhbmRsZUFkZCgpIHtcbiAgb3B0aW9ucy52YWx1ZSA9IGxpc3Rcbn1cbjwvc2NyaXB0PlxuIiwiZWxlbWVudC1wbHVzLmpzIjoiaW1wb3J0IHsgZ2V0Q3VycmVudEluc3RhbmNlIH0gZnJvbSAndnVlJ1xuaW1wb3J0IEVsZW1lbnRQbHVzIGZyb20gJ2VsZW1lbnQtcGx1cydcblxubGV0IGluc3RhbGxlZCA9IGZhbHNlXG5hd2FpdCBsb2FkU3R5bGUoKVxuXG5leHBvcnQgZnVuY3Rpb24gc2V0dXBFbGVtZW50UGx1cygpIHtcbiAgaWYgKGluc3RhbGxlZCkgcmV0dXJuXG4gIGNvbnN0IGluc3RhbmNlID0gZ2V0Q3VycmVudEluc3RhbmNlKClcbiAgaW5zdGFuY2UuYXBwQ29udGV4dC5hcHAudXNlKEVsZW1lbnRQbHVzKVxuICBpbnN0YWxsZWQgPSB0cnVlXG59XG5cbmV4cG9ydCBmdW5jdGlvbiBsb2FkU3R5bGUoKSB7XG4gIGNvbnN0IHN0eWxlcyA9IFsnaHR0cHM6Ly91bnBrZy5jb20vZWxlbWVudC1wbHVzQGxhdGVzdC9kaXN0L2luZGV4LmNzcycsICdodHRwczovL3VucGtnLmNvbS9lbGVtZW50LXBsdXNAbGF0ZXN0L3RoZW1lLWNoYWxrL2RhcmsvY3NzLXZhcnMuY3NzJ10ubWFwKChzdHlsZSkgPT4ge1xuICAgIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgICAgbGluay5ocmVmID0gc3R5bGVcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgICB9KVxuICB9KVxuICByZXR1cm4gUHJvbWlzZS5hbGwoc3R5bGVzKVxufVxuIiwidHNjb25maWcuanNvbiI6IntcbiAgXCJjb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IFwiRVNOZXh0XCIsXG4gICAgXCJqc3hcIjogXCJwcmVzZXJ2ZVwiLFxuICAgIFwibW9kdWxlXCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVSZXNvbHV0aW9uXCI6IFwiQnVuZGxlclwiLFxuICAgIFwidHlwZXNcIjogW1wiZWxlbWVudC1wbHVzL2dsb2JhbC5kLnRzXCJdLFxuICAgIFwiYWxsb3dJbXBvcnRpbmdUc0V4dGVuc2lvbnNcIjogdHJ1ZSxcbiAgICBcImFsbG93SnNcIjogdHJ1ZSxcbiAgICBcImNoZWNrSnNcIjogdHJ1ZVxuICB9LFxuICBcInZ1ZUNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogMy4zXG4gIH1cbn1cbiIsIlBsYXlncm91bmRNYWluLnZ1ZSI6IjxzY3JpcHQgc2V0dXA+XG5pbXBvcnQgQXBwIGZyb20gJy4vQXBwLnZ1ZSdcbmltcG9ydCB7IHNldHVwRWxlbWVudFBsdXMgfSBmcm9tICcuL2VsZW1lbnQtcGx1cy5qcydcbnNldHVwRWxlbWVudFBsdXMoKVxuPC9zY3JpcHQ+XG5cbjx0ZW1wbGF0ZT5cbiAgPEFwcCAvPlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcInZ1ZVwiOiBcImh0dHBzOi8vdW5wa2cuY29tL0B2dWUvcnVudGltZS1kb21AbGF0ZXN0L2Rpc3QvcnVudGltZS1kb20uZXNtLWJyb3dzZXIuanNcIixcbiAgICBcIkB2dWUvc2hhcmVkXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3VucGtnLmNvbS9lbGVtZW50LXBsdXNAbGF0ZXN0L2Rpc3QvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vZWxlbWVudC1wbHVzQGxhdGVzdC9cIixcbiAgICBcIkBlbGVtZW50LXBsdXMvaWNvbnMtdnVlXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiX28iOnt9fQ==)

- [After](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtc2VsZWN0LXYyXG4gICAgdi1tb2RlbD1cInZhbHVlXCJcbiAgICBzdHlsZT1cIndpZHRoOiAyNDBweFwiXG4gICAgbXVsdGlwbGVcbiAgICBmaWx0ZXJhYmxlXG4gICAgcmVtb3RlXG4gICAgOnJlbW90ZS1tZXRob2Q9XCJyZW1vdGVNZXRob2RcIlxuICAgIGNsZWFyYWJsZVxuICAgIDpvcHRpb25zPVwib3B0aW9uc1wiXG4gICAgOmxvYWRpbmc9XCJsb2FkaW5nXCJcbiAgICBwbGFjZWhvbGRlcj1cIlBsZWFzZSBlbnRlciBhIGtleXdvcmRcIlxuICAvPlxuICA8ZWwtYnV0dG9uIEBjbGljaz1cImhhbmRsZUFkZFwiPkNsaWNrIG1lPC9lbC1idXR0b24+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuXG5jb25zdCBzdGF0ZXMgPSBbXG4gICdBbGFiYW1hJyxcbiAgJ0FsYXNrYScsXG4gICdBcml6b25hJyxcbiAgJ0Fya2Fuc2FzJyxcbiAgJ0NhbGlmb3JuaWEnLFxuICAnQ29sb3JhZG8nLFxuICAnQ29ubmVjdGljdXQnLFxuICAnRGVsYXdhcmUnLFxuICAnRmxvcmlkYScsXG4gICdHZW9yZ2lhJyxcbiAgJ0hhd2FpaScsXG4gICdJZGFobycsXG4gICdJbGxpbm9pcycsXG4gICdJbmRpYW5hJyxcbiAgJ0lvd2EnLFxuICAnS2Fuc2FzJyxcbiAgJ0tlbnR1Y2t5JyxcbiAgJ0xvdWlzaWFuYScsXG4gICdNYWluZScsXG4gICdNYXJ5bGFuZCcsXG4gICdNYXNzYWNodXNldHRzJyxcbiAgJ01pY2hpZ2FuJyxcbiAgJ01pbm5lc290YScsXG4gICdNaXNzaXNzaXBwaScsXG4gICdNaXNzb3VyaScsXG4gICdNb250YW5hJyxcbiAgJ05lYnJhc2thJyxcbiAgJ05ldmFkYScsXG4gICdOZXcgSGFtcHNoaXJlJyxcbiAgJ05ldyBKZXJzZXknLFxuICAnTmV3IE1leGljbycsXG4gICdOZXcgWW9yaycsXG4gICdOb3J0aCBDYXJvbGluYScsXG4gICdOb3J0aCBEYWtvdGEnLFxuICAnT2hpbycsXG4gICdPa2xhaG9tYScsXG4gICdPcmVnb24nLFxuICAnUGVubnN5bHZhbmlhJyxcbiAgJ1Job2RlIElzbGFuZCcsXG4gICdTb3V0aCBDYXJvbGluYScsXG4gICdTb3V0aCBEYWtvdGEnLFxuICAnVGVubmVzc2VlJyxcbiAgJ1RleGFzJyxcbiAgJ1V0YWgnLFxuICAnVmVybW9udCcsXG4gICdWaXJnaW5pYScsXG4gICdXYXNoaW5ndG9uJyxcbiAgJ1dlc3QgVmlyZ2luaWEnLFxuICAnV2lzY29uc2luJyxcbiAgJ1d5b21pbmcnLFxuXVxuY29uc3QgbGlzdCA9IHN0YXRlcy5tYXAoKGl0ZW0pOiBMaXN0SXRlbSA9PiB7XG4gIHJldHVybiB7IHZhbHVlOiBgdmFsdWU6JHtpdGVtfWAsIGxhYmVsOiBgbGFiZWw6JHtpdGVtfWAgfVxufSlcblxuaW50ZXJmYWNlIExpc3RJdGVtIHtcbiAgdmFsdWU6IHN0cmluZ1xuICBsYWJlbDogc3RyaW5nXG59XG5cbmNvbnN0IHZhbHVlID0gcmVmKFsndmFsdWU6VGV4YXMnXSlcbmNvbnN0IG9wdGlvbnMgPSByZWY8TGlzdEl0ZW1bXT4oW10pXG5jb25zdCBsb2FkaW5nID0gcmVmKGZhbHNlKVxuXG5jb25zdCByZW1vdGVNZXRob2QgPSAocXVlcnk6IHN0cmluZykgPT4ge1xuICBpZiAocXVlcnkgIT09ICcnKSB7XG4gICAgbG9hZGluZy52YWx1ZSA9IHRydWVcbiAgICBzZXRUaW1lb3V0KCgpID0+IHtcbiAgICAgIGxvYWRpbmcudmFsdWUgPSBmYWxzZVxuICAgICAgb3B0aW9ucy52YWx1ZSA9IGxpc3QuZmlsdGVyKChpdGVtKSA9PiB7XG4gICAgICAgIHJldHVybiBpdGVtLmxhYmVsLnRvTG93ZXJDYXNlKCkuaW5jbHVkZXMocXVlcnkudG9Mb3dlckNhc2UoKSlcbiAgICAgIH0pXG4gICAgfSwgMjAwKVxuICB9IGVsc2Uge1xuICAgIG9wdGlvbnMudmFsdWUgPSBbXVxuICB9XG59XG5cbmZ1bmN0aW9uIGhhbmRsZUFkZCgpIHtcbiAgb3B0aW9ucy52YWx1ZSA9IGxpc3Rcbn1cbjwvc2NyaXB0PlxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctMTczNjItZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXSxcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWUsXG4gICAgXCJhbGxvd0pzXCI6IHRydWUsXG4gICAgXCJjaGVja0pzXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5pbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgY29uc3Qgc3R5bGVzID0gWydodHRwczovL3ByZXZpZXctMTczNjItZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnLCAnaHR0cHM6Ly9wcmV2aWV3LTE3MzYyLWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ10ubWFwKChzdHlsZSkgPT4ge1xuICAgIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgICAgbGluay5ocmVmID0gc3R5bGVcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgICB9KVxuICB9KVxuICByZXR1cm4gUHJvbWlzZS5hbGwoc3R5bGVzKVxufSIsIl9vIjp7InNob3dIaWRkZW4iOnRydWUsInN0eWxlU291cmNlIjoiaHR0cHM6Ly9wcmV2aWV3LTE3MzYyLWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)

Reply:
#16714 changes the focus behavior of el-select-v2. Before #16714, `activeElement` value is `input`, After it is `wrapperRef`. So..., just corrent the focus to input, it should be solved.

Reply:
I took a closer look and found that it was indeed the problem caused by #16714.

before focus() –> toggleMenu()
after toggleMenu() -> focus()

IMO, Perhaps you can modify the `useFocusController` function to ensure that the `focus` is triggered first.

```diff
- useEventListener(wrapperRef, 'click', handleClick)
+ useEventListener(wrapperRef, 'click', handleClick, true)
```

Reply:
@tolking ACK! It works, thanks for your advice, review pls.